src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -955,20 +955,11 @@
     <PackageReference Include="System.Collections.Immutable" />
   </ItemGroup>
 
-  <ItemGroup>
-    <!-- Repotools will add a PackageReference for the compilers package based on the current runtime.
```

Comment:
@AndyGerlicher This didn't appear to be the case in my testing; how did you figure out that it was?

Reply:
This file adds the PackageReference:
https://github.com/dotnet/roslyn-tools/blob/master/src/RepoToolset/tools/Compiler.props

And it gets imported here:
https://github.com/dotnet/roslyn-tools/blob/8e0ee77c949d0c8f42ecbaa3ccd6d382111519c4/src/RepoToolset/tools/Settings.props#L18

But only based on `$(UsingToolMicrosoftNetCompilers)`. Which we set to `false` in our repo. And that's ignored and set to `true` here:
https://github.com/dotnet/roslyn-tools/blob/145e3b3d376ce385f40dfc1ef24dbefaec6b3b6e/src/RepoToolset/tools/DefaultVersions.props#L34

But only when `$(CIBuild)` is `true`, which is set here:
https://github.com/Microsoft/msbuild/blob/master/build/build.ps1#L239

When you pass -ci to the build.ps1. Which is why this PR worked on your machine but will likely fail in some weird way if VSTS eventually picks it up.

It's so straightforward it really almost explains itself.

Reply:
In all seriousness though I would love to disable that entirely and do it explicitly when it's needed like you've got it. This was a nightmare.

Oh and I remembered the reason this worked before is because it didn't matter that we were adding a PR twice (once in repo tools once here). It matters now because CLI emits a warning if there are any two package references that are `IsImplicitlyDefined=true`.